sdk.mjs

```diff
@@ -115,7 +115,7 @@ export const htmlFriendlyToTitle = (htmlFriendlyTitle) =>
     .replace(/&#([0-9]+);/g, (_, num) => String.fromCodePoint(parseInt(num)))
     .replace(
       /&(quot|amp|lt|gt);/g,
-      (_, ref) => ({ quot: '"', amp: '&', lt: '<', gt: '>' }[ref]),
+      (_, ref) => ({ quot: '"', amp: '&', lt: '<', gt: '>' })[ref],
```

Comment:
I think that this change was done by a previous version of Prettier. We commited it a few weeks ago and then failed again. Could you reformat?